src/Translator/TranslatorFactory.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Validator\Translator;
+
+use Laminas\I18n\Translator\LoaderPluginManager;
+use Laminas\I18n\Translator\Translator as I18nTranslator;
+use Laminas\I18n\Translator\TranslatorInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+use Laminas\ServiceManager\ServiceManager;
+use Psr\Container\ContainerInterface;
+use Traversable;
+
+use function array_key_exists;
+use function assert;
+use function extension_loaded;
+use function is_array;
+
+/**
+ * Overrides the translator factory from the i18n component in order to
+ * replace it with the bridge class from this namespace.
+ */
+class TranslatorFactory implements FactoryInterface
```

Comment:
```suggestion
final class TranslatorFactory
```

Final by default, also, I can't see any reason to implement `FactoryInterface` unless I'm missing something?